utils/ansible/ftest/vars/Rocky8.yml

```diff
@@ -6,39 +6,68 @@ daos_openmpi_module_path: /usr/share/modulefiles/mpi/openmpi-x86_64
 daos_base_deps:
   rpms:
     - bash
+    - capstone
     - gdb
     - glibc
     - jq
     - lbzip2
-    - mercury
+    - numactl
+    - numactl-libs
+    - numatop
     - platform-python
     - python36
     - systemd
     - tree
 
 daos_dev_deps:
```

Comment:
Discussed off-line but are these packages needed to build daos or to run the tests?  `argobots-devel` I would assume is for building but a lot are clearly for running the tests.  https://github.com/daos-stack/daos/pull/13565 is relevant to this work and what would be best is if new newly added `requirements-ftest.txt` file from that PR could be used here.

Reply:
I will split this PR in two: one PR for ansible stuff and one PR for the functional test.
I will update the ansible PR according to your remarks:

- [ ] split the PR

Reply:
> I will split this PR in two: one PR for ansible stuff and one PR for the functional test. I will update the ansible PR according to your remarks:
> 
> * [x]  split the PR

Fixed with commit bc26fdcb76c98c3647a786b188f7c6b949668ba4

Update of the ansible script will be done in the ticket [DAOS-15603](https://daosio.atlassian.net/browse/DAOS-15603)